LSP.scala

```diff
@@ -9,50 +9,33 @@ import fs2.io.file.Path
 import cats.parse.Parser
 import QuickmaffsCompiler.CompileError
 import QuickmaffsCompiler.Index
-import langoustine.lsp.RuntimeBase.DocumentUri
-import langoustine.lsp.RuntimeBase.uinteger
+import langoustine.lsp.runtime.DocumentUri
+import langoustine.lsp.runtime.Opt
+import langoustine.lsp.runtime.uinteger
 
 import cats.syntax.all.*
+import langoustine.lsp.app.LangoustineApp
 
-object LSP extends IOApp.Simple:
+object LSP extends LangoustineApp:
```

Comment:
There's even a [`LangoustineApp.Simple`](https://neandertech.github.io/langoustine/api/langoustine/lsp/app/LangoustineApp$$Simple.html)!

Reply:
I know, but I wanted to keep the `IO.println` on finalization - which required something like a Resource